docs/pages/helm/guides/getting-started.mdx

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
This is really a helm reference, let's move it to Helm Refernce inside Kubernetes access.